docs/tutorial/fused_rings.rst

```diff
@@ -0,0 +1,59 @@
+Atom Mapping and Fused Rings
+-----------------------------
+
+Creating atom mappings between two ring systems where only one is fused can be difficult as there are two possible
+options. ``Kartograf`` is able to create both possible mappings and the user can select which via the use of the
+``allow_partial_fused_rings`` flag. In this short tutorial we will demonstrate how to use this flag to generate the desired mappings
+for this special case, first lets start by creating our example molecules and aligning them, we will be following the
+:ref:`mapping tutorial <mapping_tutorial>` for this part.
+
+.. code-block::
+
+    from rdkit import Chem
+    from kartograf import SmallMoleculeComponent
+    from kartograf.atom_aligner import align_mol_shape
+
+    # Preprocessing from Smiles - Here you can add your Input!
+    smiles = ["c1ccccc1-c2ccccc2", "C1Cc2cc(-c3ccccc3)ccc2C1"]
+    rdmols = [Chem.MolFromSmiles(s) for s in smiles]
+    rdmols = [Chem.AddHs(m, addCoords=True) for m in rdmols]
+    [Chem.rdDistGeom.EmbedMolecule(m, useRandomCoords=False, randomSeed = 0) for m in rdmols]
+
+    # Build Small Molecule Components
+    molA, molB = [SmallMoleculeComponent.from_rdkit(m) for m in rdmols]
+
+    # Align the mols first
+    a_molB = align_mol_shape(molB, ref_mol=molA)
+
+The first type of mapping would be to keep the common rings fixed and grow the second fused ring using dummy atoms,
+this approach would involve transforming fewer atoms but can lead to errors in the calculated free energies [1]_.
+This is the default behaviour of the ``KartografAtomMapper``:
+
+.. code-block::
+
+    from kartograf import KartografAtomMapper
+    mapper = KartografAtomMapper()
+    # Get Mapping
+    kartograf_mapping = next(mapper.suggest_mappings(molA, a_molB))
+    kartograf_mapping
+
+.. image:: ../_static/img/fused_ring_partial.png
+
+The second option considers the entire fused ring system as a unique set of atoms and is normally preferred however,
```

Comment:
Maybe add something that the statistical error can be large (given a fixed sampling time). I think the reason why we currently don't want this as default is that the systematic error might be smaller than the statistical one (of the larger transformation). TBD depending on the benchmarking study =)

Reply:
Yeah good point added a bit on the statistical error, and then hopefully we can update this again after the benchmark!